src/NServiceBus.Core/Recoverability/Faults/ErrorQueueSettings.cs

```diff
@@ -23,19 +29,31 @@ public static string ErrorQueueAddress(this ReadOnlySettings settings)
         {
             string errorQueue;
 
-            if (settings.TryGet("errorQueue", out errorQueue))
+            if (TryGetErrorQueueAddress(settings, out errorQueue))
             {
-                Logger.Debug("Error queue retrieved from code configuration via 'EndpointConfiguration.SendFailedMessagesTo().");
                 return errorQueue;
             }
 
+            return DefaultErrorQueueName;
+        }
+
+        internal static bool TryGetErrorQueueAddress(this ReadOnlySettings settings, out string errorQueue)
```

Comment:
This being internal means that MSMQ won't be able to call it when it's moved out of core. Seems like we should go ahead and account for this now. Do we just want to make this public, or do we need to come up with a different approach for MSMQ to get this information?